### ISSUE TITLE
N1MM Contact Broadcast does not deliver tx- and rxfrequency

### ISSUE THREAD

**Issue by dg9vh** [mbridak/not1mm#126]
**Describe the bug**
When using the N1MM Data-Export by configuration in settings, the delivered xml does not contain frequencies but only bands.

**To Reproduce**
1. Go to configuration settings/N1MM
2. Enable Send N1MM Packets and activate "Contact"
3. Log QSO
4. inspect resulting xml received by Client.

**Expected behavior**
XML should contain the logged qrgs in rx und tx.

**Screenshots**
![image](https://github.com/user-attachments/assets/c1c92f18-3398-4506-a19d-8eb7b6305e03)

![image](https://github.com/user-attachments/assets/3bd0280b-ee1f-4635-a83b-4c504c108078)

**Desktop (please complete the following information):**
 - OS: Debian Linux
 - Version 12

**Comment by mbridak** [comment]
Thanks for the bug report. I just pushed a fix.

**Comment by dg9vh** [comment]
Hi, tnx, will check this this evening - cause is how I found this, I am actually building a service that listens for the logs of N1MM / Not1MM and pushes the QSOs as ADIF-entries into Cloudlog / Wavelog online log.

We used your program this weekend in IARU-R1 Fieldday (by using a different Contest-layout because R1 Fieldday was not in the list)... it worked good so far and I decided, to use it on my Linux-environment as primary contest logger in the future. Therefore I needed this service I develloped but got no QRGs in your data :-)

Will close this issue, if solved. So far,
best 73
Kim
